apps/src/templates/progress/LessonLock.jsx

```diff
@@ -52,7 +51,8 @@ const styles = {
   button: {
     paddingLeft: 0,
     paddingRight: 0,
-    width: '100%'
+    width: '100%',
+    margin: '5px 0px 0px 0px'
```

Comment:
I list it this way so it overrides a button margin styling that uses 5 px everywhere

Reply:
Can you add a comment to the code so we don't accidentally change it and stop overriding that style?